internal/hcsoci/resources.go

```diff
@@ -7,6 +7,18 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// MountType represents the type of mount being added
+type MountType = string
+
+// MountType const
+const (
+	// supported internal mount types
+	Bind                  MountType = "bind"
```

Comment:
Can you prefix the constants with `MountType` so they are more clearly namespaced?